<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetOfStart</class>
 <widget class="QWidget" name="widgetOfStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="groupOfButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonOfQuestion">
        <property name="text">
         <string>答题页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOfSelect">
        <property name="text">
         <string>选择页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOfCollect">
        <property name="text">
         <string>收藏页</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackOfWindows">
     <property name="styleSheet">
      <string notr="true">#stackOfWindows{border: 5px solid black}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageOfQuestion">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="3">
        <widget class="QScrollBar" name="scrollOfQuestion">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="textOfQuestion">
         <property name="text">
          <string>1. 题目标题</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="fowardButton">
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLabel" name="sumOfQuestions">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="indexOfCurrentQuestion">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>题号显示</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="optionOfQuestion">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="optionOfA">
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="textOfA">
            <property name="text">
             <string>选项 A</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="optionOfB">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="textOfB">
            <property name="text">
             <string>选项 B</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="optionOfC">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="textOfC">
            <property name="text">
             <string>选项 C</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="optionOfD">
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="textOfD">
            <property name="text">
             <string>选项 D</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>→</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="collectButton">
         <property name="text">
          <string>☆/⭐</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOfSelect">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QLabel" name="textOfPath">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="titleOfPath">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="addPathButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QTableWidget" name="tableOfQuestionType"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOfCollect"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
